src/components/frame/Extensions/Header/Header.tsx

```diff
@@ -1,57 +1,82 @@
 import clsx from 'clsx';
 import React from 'react';
 
+import { useIntl } from 'react-intl';
```

Comment:
Move the import package to the top (imports packages and other packages like hooks, types,components etc. should be separate by an empty line).

Reply:
Fixed